tutorials/applications/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,1437 @@
+{
```

Comment:
<p>Remains to add an "Introduction" section explaining what this notebook will show. What does "SPAM" mean (briefly)? Same for each type of noise</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Added a more thorough introduction.